### ISSUE TITLE
Application customizer header rendering over document preview

### ISSUE THREAD

**Issue by rob-windsor** [SharePoint/sp-dev-docs#2877]
#### Category
- [ ] Question
- [ ] Typo
- [x] Bug
- [ ] Additional article idea

#### Expected or Desired Behavior

The application customizer extension should not render a header or footer over top of a document preview.

#### Observed Behavior

The application customizer extension renders a header over the document preview. The screen capture below shows the menu, rendered by an application customizer extension, being displayed over top of a document preview.

![Document Preview](https://user-images.githubusercontent.com/5740654/47971646-84d80800-e061-11e8-8468-c39aad648f1a.JPG)

#### Steps to Reproduce

Open the preview for an office document in any site that renders a header using an application customizer extension.

**Comment by jxnx888** [comment]
Hi @rob-windsor  , The GIF below is the test from my side. I cannot reproduce your issue. Could you please provide more detail?
 
![test](https://user-images.githubusercontent.com/22604980/48114761-42017480-e29b-11e8-90e8-3b6ee041da3d.gif)

BTW, I noticed the icon in your screenshot, the word is still in Sharepoint website. In my end, when I open a word, it will open it in a new tab automatically.

**Comment by rob-windsor** [comment]
Do not open the document for editing. Instead select an Office document in a document library, open the context (ECB) menu for the document, and select Preview.

**Comment by jxnx888** [comment]
@rob-windsor I still cannot reproduce your issue. Could you please provide more detail? Thanks.

![appcusextension](https://user-images.githubusercontent.com/22604980/48174679-f5be3f00-e343-11e8-8f61-8f38a6d88785.gif)

**Comment by rob-windsor** [comment]
@jxnx888 I went through the documentation again and noticed the use of the `this.context.placeholderProvider.changedEvent` in the `onInit` method. I haven't encountered that event  before and, as such, I'm not using it in my extension. I'll try refactoring my code to see if adding the use of the `changedEvent` addresses my issue.

**Comment by jxnx888** [comment]
@rob-windsor Ok, great! Please let me know if you have any update.

**Comment by rob-windsor** [comment]
@jxnx888  I figured it out. It was not that I wasn't using the changedEvent event, it was instead a z-index issue. To reproduce, modify the header rendering code from the [Use page placeholders](https://docs.microsoft.com/en-us/sharepoint/dev/spfx/extensions/get-started/using-page-placeholder-with-extensions) tutorial to look like this:

```
if (this._topPlaceholder.domElement) {
  this._topPlaceholder.domElement.innerHTML = ` 
    <div class="${styles.app}">
        <div class="${styles.top}">
            <i class="ms-Icon ms-Icon--Info" aria-hidden="true"></i>
            <span style="color:black;z-index:1000">${escape(topString)}</span>
        </div>
    </div>`;
}
```

Now when you follow the steps I described above you will see the header text displaying above the document preview. 

![Document preview](https://user-images.githubusercontent.com/5740654/48302464-5461f500-e4cb-11e8-8f5d-5b3e5c95178a.png)

Thank you very much for your help in this matter.
